src/bootstrap-wysiwyg.js

```diff
@@ -205,13 +206,18 @@
 				editor.focus();
 				$.each(files, function (idx, fileInfo) {
 					if (/^image\//.test(fileInfo.type)) {
-						$.when(readFileIntoDataUrl(fileInfo)).done(function (dataUrl) {
-							underscoreThrottle(execCommand('insertimage', dataUrl), options.keypressTimeout);
-							editor.trigger('image-inserted');
-						}).fail(function (e) {
-							options.fileUploadError("file-reader", e);
-						});
-					} else {
+						if (fileInfo.size>maxFileSize){
```

Comment:
@ezekieltanch would you be up to working on this? If not then I'd be happy to look into it sometime soon.